Server/Config/db.js

```diff
@@ -0,0 +1,12 @@
+import { Pool } from 'pg';
+
+const connectionString = 'postgresql://postgres:123@localhost:5432/mydiary_db';
```

Comment:
I forgot to mention this. You need to put this database url in your `.env` file.
this should look like this `
const connectionString = process.env.DATABASE_URL;
`

Reply:
I made some changes.